sonar-scanner-engine/src/main/java/org/sonar/scanner/ProjectAnalysisInfo.java

```diff
@@ -52,23 +54,22 @@ public String analysisVersion() {
   }
 
   private Date loadAnalysisDate() {
+    Optional<String> value = settings.get(CoreProperties.PROJECT_DATE_PROPERTY);
+    if (!value.isPresent()) {
+      return new Date(system2.now());
+    }
     Date date;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "date" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1481)